.goreleaser.yml

```diff
@@ -87,6 +87,16 @@ release:
 
     For key updates, see the [changelog](https://golangci-lint.run/product/changelog/#{{ .Major }}{{ .Minor }}{{ .Patch }}).
 
+signs:
+  - signature: ${artifact}.cosign.bundle
+    cmd: cosign
```

Comment:
I guess some docs how to verify downloads with cosign would not hurt. But we don't have any for verifying the sha256sums either, so not sure. https://github.com/golangci/golangci-lint/pull/5794 contains changes for verifying in the installer script.